src/kernel/src/ucred/mod.rs

```diff
@@ -52,6 +52,14 @@ impl Ucred {
         self.auth.caps.is_system()
     }
 
+    pub fn is_unk1(&self) -> bool {
+        self.auth.caps.is_unk1() || self.auth.attrs.is_unk1()
```

Comment:
I think this one required both condition to be true instead of either one is true.